notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -8,7 +8,8 @@
     "**Learning Objectives:**\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        # Feature: pickup_longitude, pickup_latitude, dropoff_longitude, dropoff_latitude
</pre><p>nit: "Features" or "Feature columns" in plural</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>updated</p>